pkg/trace/sampler/sampler.go

```diff
@@ -53,6 +53,10 @@ const (
 // SamplingPriority is the type encoding a priority sampling decision.
 type SamplingPriority int8
 
+// SamplingMechanism this type is an alias for pkg/proto/datadog/trace/remote_rates.proto
```

Comment:
```suggestion
// SamplingMechanism is an alias for pkg/proto/datadog/trace/remote_rates.proto
```

This also removes some useful-looking comments from the original definition, we should probably keep them.